examples/tuning/cta_scdeepsort/main.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import pprint
+import sys
+from pathlib import Path
+from typing import get_args
+
+import numpy as np
+import torch
+import wandb
+
+from dance import logger
+from dance.datasets.singlemodality import CellTypeAnnotationDataset
+from dance.modules.single_modality.cell_type_annotation.scdeepsort import ScDeepSort
+from dance.pipeline import PipelinePlaner, get_params, get_step3_yaml, run_step3, save_summary_data
+from dance.typing import LogLevel
+from dance.utils import set_seed
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--batch_size", type=int, default=500)
+    parser.add_argument("--cache", action="store_true", help="Cache processed data.")
+    # parser.add_argument("--dense_dim", type=int, default=400, help="number of hidden gcn units")
```

Comment:
There are hyperparameters that both the preprocessing function and the model depend on. Delete them and read them through the configuration file.